pallets/issue/src/tests.rs

```diff
@@ -434,3 +480,202 @@ fn test_set_issue_period_only_root() {
 		assert_ok!(Issue::set_issue_period(RuntimeOrigin::root(), 1));
 	})
 }
+
+#[test]
+fn test_request_issue_fails_exceed_limit_volume_for_issue_request() {
+	run_test(|| {
+		crate::Pallet::<Test>::_rate_limit_update(
+			std::option::Option::<u128>::Some(1u128),
+			DEFAULT_COLLATERAL_CURRENCY,
+			7200u64,
+		);
+
+		let issue_amount = 3;
```

Comment:
I would change the `issue_amount` to `2` because it should already fail for 2.

Reply:
agree!